src/containers/WelcomePage/components/SavedSearch.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { FC, useState, useEffect } from 'react';
+import BEMHelper from 'react-bem-helper';
+import { Link } from 'react-router-dom';
+import queryString from 'query-string';
+
+import { injectT } from '@ndla/i18n';
+import { DeleteForever } from '@ndla/icons/editor';
+import Tooltip from '@ndla/tooltip';
+
+import { TranslateType } from '../../../interfaces';
+import IconButton from '../../../components/IconButton';
+import { fetchSubject, fetchResourceType } from '../../../modules/taxonomy';
+
+interface Props {
+  deleteSearch: Function;
+  locale: string;
+  search: string;
+  index: number;
+  t: TranslateType;
+}
+
+export const classes = new BEMHelper({
+  name: 'saved-search',
+  prefix: 'c-',
+});
+
+const SavedSearch: FC<Props> = ({ deleteSearch, locale, search, index, t }) => {
+  const [subjectName, setSubjectName] = useState('');
+  const [resourceTypeName, setResourceTypeName] = useState('');
+
+  const searchObject = queryString.parse(search);
+  const subject = searchObject['subjects'] || '';
+  const resourceType = searchObject['resource-types'] || '';
+
+  useEffect(() => {
+    if (subject) {
+      fetchSubjectName(subject, locale);
+    }
+    if (resourceType) {
+      fetchResourceTypeName(resourceType, locale);
+    }
+  }, [subject, resourceType]);
+
+  const fetchSubjectName = async (id: string, locale: string) => {
+    const result = await fetchSubject(id, locale);
+    setSubjectName(result.name);
+  };
+
+  const fetchResourceTypeName = async (id: string, locale: string) => {
+    const result = await fetchResourceType(id, locale);
```

Comment:
Denne tar ikkje høgde for innholdstype Emne, som er spesialhåndtert i søket. Dersom du lagrer et søk med emne som innholdstype vises det som tom streng i lista.

Reply:
Tror det skal virke rett nå, men var det andre context typer enn Emne? (mtp. hvilken tekst som skal vises)